update/v1alpha1/doc.go

```diff
@@ -0,0 +1,8 @@
+// +k8s:deepcopy-gen=package,register
+// +k8s:defaulter-gen=TypeMeta
+// +k8s:openapi-gen=true
+
+// +kubebuilder:validation:Optional
```

Comment:
Don't do this, it is not the default and is likely to cause confusion if someone doesn't add an explicit required marker.

Reply:
Also copy pasta from `config/v1alpha1`. Will fix, unsure what does it cause.